services/nycrc/nycrc.tester.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const { isIntegerPercentage } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('valid .nycrc')
+  .get('/yargs/yargs.json?config=.nycrc')
+  .intercept(nock =>
+    nock('https://api.github.com')
+      .get('/repos/yargs/yargs/contents/.nycrc?ref=master')
+      .reply(200, {
+        content: Buffer.from(
+          JSON.stringify({
+            branches: 89,
+            lines: 85,
+          })
+        ).toString('base64'),
+        encoding: 'base64',
+      })
+  )
+  .expectBadge({ label: 'min coverage', message: isIntegerPercentage })
+
+t.create('.nycrc with no thresholds')
+  .get('/yargs/yargs.json?config=.nycrc')
+  .intercept(nock =>
+    nock('https://api.github.com')
+      .get('/repos/yargs/yargs/contents/.nycrc?ref=master')
+      .reply(200, {
+        content: Buffer.from(
+          JSON.stringify({
+            reporter: 'foo',
+          })
+        ).toString('base64'),
+        encoding: 'base64',
+      })
+  )
+  .expectBadge({
+    label: 'min coverage',
+    message: '"branches" or "lines" threshold missing',
+  })
+
+t.create('package.json with nyc stanza')
```

Comment:
Would a test for the `no nyc or c8 stanza found` case be useful? You could simply use _badges/shields_ as an example for this.

Reply:
Done, used `badges/shields` like recommended.